docs/includes/apiargs-MongoDBCollection-method-explain-param.yaml

```diff
@@ -0,0 +1,13 @@
+arg_name: param
+name: $explainable
+type: :phpclass:`MongoDB\\Operation\\Explainable`
+description: |
+  The command to explain.
```

Comment:
Let's add a new paragraph here:

```
  Possible commands include, but are not limited to:

  - :phpclass:`MongoDB\\Operation\\Count`
  - :phpclass:`MongoDB\\Operation\\DeleteMany`
  - :phpclass:`MongoDB\\Operation\\DeleteOne`
  - :phpclass:`MongoDB\\Operation\\Distinct`
  - :phpclass:`MongoDB\\Operation\\Find`
  - :phpclass:`MongoDB\\Operation\\FindOne`
  - :phpclass:`MongoDB\\Operation\\FindOneAndDelete`
  - :phpclass:`MongoDB\\Operation\\FindOneAndReplace`
  - :phpclass:`MongoDB\\Operation\\FindOneAndUpdate`
  - :phpclass:`MongoDB\\Operation\\UpdateMany`
  - :phpclass:`MongoDB\\Operation\\UpdateOne`
```

If this renders poorly within the parameter table, we can consider moving this to its own section elsewhere on the method page.

I'm using `:phpclass:` for consistency, even though these won't render as links since we have no API pages for the classes.

Reply:
It did not render well within the parameter table so I've moved it to the end of the definition section, but I can move it to its own "Explainable Commands" section if you think that would make more sense.

Reply:
A separate "Explainable Commands" section on the main method page makes sense to me. We can position it in the same place as the optional "Behavior" section on some other methods -- after return values and exceptions and before the example.